frontend/lib/src/components/widgets/FileUploader/FileDropzoneInstructions.tsx

```diff
@@ -51,7 +51,7 @@ const FileDropzoneInstructions = ({
         {acceptedExtensions.length
           ? ` • ${acceptedExtensions
               .join(", ")
-              .replace(/\./g, "")
+              .replace(/\./, "")
```

Comment:
Hm, I don't think this is quite what we want here either --

In the case where the `type` argument of a  `st.file_uploader` is passed multiple extensions, only the first of the extensions will have the leading dot removed, where we'd want to do so for each extension. We probably want to remove the leading dot for each extension before joining them

Reply:
Yes, makes sense. I adapted the code accordingly.